components/builder-web/app/shared/dialog/job-cancel/job-cancel.dialog.html

```diff
@@ -5,16 +5,16 @@ <h1>Confirm Job Cancellation</h1>
   </section>
   <section class="body">
     <p>
-      Completed build jobs will remain in the <b>unstable</b> channel. Pending jobs will not be processed.
+      Completed packages will remain in the <b>unstable</b> channel. Remaining jobs will be canceled or terminated.
```

Comment:
Not sure if there is value in calling out "cancelled or terminated"?  Maybe just say "Remaining jobs will be canceled" and leave it at that?

Reply:
Ok. I just noticed that cancellation also seems to stop individual jobs that are currently running, so I figured it might be good (or not bad) to mention that also.